assets/src/components/activities/DeliveryElement.tsx

```diff
@@ -63,6 +63,11 @@ export interface DeliveryElementProps<T extends ActivityModelSchema> {
     partAttemptGuid: string,
     response: StudentResponse,
   ) => Promise<Success>;
+  onInitPart: (
```

Comment:
we need to make this optional, or else the other Torus activities will have errors.